### ISSUE TITLE
20:04 Docker images are pulling updates from Xenial repo

### ISSUE THREAD

**Issue by VeCAD** [vistart/Dockerfiles#3]
Hi, I frequently use your Docker images, the 20.04 version recently have been failing builds. upon checking further they seem to be pulling updates from Xenial repo instead of Focal including the latest cuda/cudnn tags

```
FROM vistart/cuda:10.2-cudnn7-devel-ubuntu20.04
.......
Get:1 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial-updates/main amd64 libpython3.5-minimal amd64 3.5.2-2ubuntu0~16.04.13 [524 kB]
Get:2 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial-updates/main amd64 python3.5-minimal amd64 3.5.2-2ubuntu0~16.04.13 [1597 kB]
Get:3 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial/main amd64 python3-minimal amd64 3.5.1-3 [23.3 kB]

```

**Comment by vistart** [comment]
This problem has been located, the version of the ubuntu mirror released by Docker Official is confusing, and the mirror is currently being remade. thank you for your reply!

**Comment by vistart** [comment]
> Hi, I frequently use your Docker images, the 20.04 version recently have been failing builds. upon checking further they seem to be pulling updates from Xenial repo instead of Focal including the latest cuda/cudnn tags
> 
> ```
> FROM vistart/cuda:10.2-cudnn7-devel-ubuntu20.04
> .......
> Get:1 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial-updates/main amd64 libpython3.5-minimal amd64 3.5.2-2ubuntu0~16.04.13 [524 kB]
> Get:2 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial-updates/main amd64 python3.5-minimal amd64 3.5.2-2ubuntu0~16.04.13 [1597 kB]
> Get:3 http://mirrors.tuna.tsinghua.edu.cn/ubuntu xenial/main amd64 python3-minimal amd64 3.5.1-3 [23.3 kB]
> ```

`10.2-*-ubuntu20.04` has been deprecated. Nvidia has not released cuda10 related packages for ubuntu 20.04. Please use `10.2-*-ubuntu18.04` or `11.*-*-ubuntu20.04` instead.

**Comment by VeCAD** [comment]
yes, I am reverting to official nvidia docker image
FROM nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04
